registry/api/errcode/register.go

```diff
@@ -33,14 +33,25 @@ var (
 		HTTPStatusCode: http.StatusMethodNotAllowed,
 	})
 
+	// ErrorCodeUnauthenticated is returned if a request requires
+	// authentication.
+	ErrorCodeUnauthenticated = Register("errcode", ErrorDescriptor{
+		Value:   "UNAUTHENTICATED",
+		Message: "authentication required",
+		Description: `The access controller was unable to authenticate
+		the client. Often this will be accompanied by a
+		Www-Authenticate HTTP response header indicating how to
+		authenticate.`,
+		HTTPStatusCode: http.StatusUnauthorized,
+	})
+
 	// ErrorCodeUnauthorized is returned if a request is not authorized.
 	ErrorCodeUnauthorized = Register("errcode", ErrorDescriptor{
 		Value:   "UNAUTHORIZED",
```

Comment:
@jlhawn Please change this to match the table above. There is no need to have "unauthenticated" and "unauthorized". This diff should merely add a "denied".